cyhy/core/geoloc.py

```diff
@@ -39,3 +39,15 @@ def lookup(self, ip):
             return (response.location.longitude, response.location.latitude)
         except AddressNotFoundError:
             return (None, None)
+
+    def check_restricted_cidr(self, cidr):
+        restricted_countries = ["China","Iran","North Korea","Russia"]
```

Comment:
The `restricted_countries` variable should be defined at the top of the file as `RESTRICTED_COUNTRIES`, similar to what is done with `GEODB_FILES`.